### ISSUE TITLE
Replace gifs with better quality gifs and optimise the images 

### ISSUE THREAD

**Issue by Eldenroot** [WhiteNeo/post_rate#62]
Replace gifs with better quality gifs and optimise the images

**Comment by WhiteNeo** [comment]
Yeah this is the next move on :) i will do it this week end this wold be open until it is covered up.

**Comment by WhiteNeo** [comment]
Someone have better images i can not found better images this time, sorry.
